src/main/java/org/apache/commons/lang3/reflect/TypeUtils.java

```diff
@@ -1194,9 +1207,10 @@ public static boolean isInstance(final Object value, final Type type) {
      * @return a non-empty array containing the lower bounds of the wildcard
      * type.
      */
-    public static Type[] getImplicitLowerBounds(final WildcardType wildcardType) {
+    public static Type @MinLen(1) [] getImplicitLowerBounds(final WildcardType wildcardType) {
         Validate.notNull(wildcardType, "wildcardType is null");
-        final Type[] bounds = wildcardType.getLowerBounds();
+        @SuppressWarnings("assignment.type.incompatible") // Validate.notNull() => wildcardType.getImplicitLowerBounds() returns an array of @MinLen(1)
+        final Type @MinLen(1) [] bounds = wildcardType.getLowerBounds();
```

Comment:
Same as above: It looks like the `bounds` local variable can be an empty array and therefore should not be annotated `@MinLen(1)`.